src/services/shortLink/test/fixtures.ts

```diff
@@ -0,0 +1,126 @@
+import { ClientHostManager, Context, HttpMethod, PermissionLevel } from '@graasp/sdk';
+
+import { SHORT_LINKS_LIST_ROUTE, SHORT_LINKS_ROUTE_PREFIX } from '../../../utils/constants';
+import { setItemPublic } from '../../item/plugins/itemTag/test/fixtures';
+import { getDummyItem } from '../../item/test/fixtures/items';
+import {
+  saveItemAndMembership,
+  saveMembership,
+} from '../../itemMembership/test/fixtures/memberships';
+import { Member } from '../../member/entities/member';
+
+export const MOCK_ALIAS = 'mocked-alias';
+export const MOCK_PLATFORM = Context.Player;
+export const MOCK_ITEM_ID = '1c21f7f8-1917-4dfc-82b6-21d7136812e8';
+
+export const mockShortLinkPayload = ({
+  alias,
+  platform = MOCK_PLATFORM,
+  itemId,
+}: {
+  alias: string;
+  platform?: Context;
+  itemId: string;
+}) => {
+  return {
+    alias,
+    platform,
+    item: { id: itemId },
```

Comment:
Wouldn't it be clearer if we use `itemId` instead of `item: {id}`?

Reply:
The payload returned by this function is used in an API POST call. The body of the API must contain `item: {id}` so I think it is simpler to return the payload as it must be send to the API. But maybe I missed something ?